harvest/trader/trader.py

```diff
@@ -246,22 +248,11 @@ async def main(self, df_dict, timestamp, flush: bool=False):
 
     def main_helper(self, df_dict):
 
-        new_day = False
-        # Init queue on a new day 
-        if self.timestamp.date() > self.timestamp_prev.date():
-            debug("Initializing queue...")
-            self._queue_init(self.fetch_interval) 
-            new_day = True
```

Comment:
@tfukaza  Why do we need to restart the queue on a new day?

Reply:
This was a failsafe in case the aggregation of candles did not work. I think it can be removed like you did, since aggregation seems to be working fine